### DISCUSSION TITLE
SO training and labs

### DISCUSSION THREAD

**Discussion by Carlos-mb** [Security-Onion-Solutions/securityonion#13550] [General]
Hi all
I have been searching for Security Onion (SO), SIEM, SOC training.
I have found some webs that offers training for learning paths like SOC Analyst and SO.
Before I make a decision Id like to know your experience and valuable opinion.

Do you recommend any site for this?

It should be great to find a site with SO update training and labs

Thanks!

**Comment by dougburks** [comment]
We recommend the official Security Onion training at:
https://securityonion.com/training
